pkg/version/check_test.go

```diff
@@ -83,9 +76,9 @@ func (s *checkSuite) TestCheckClusterVersion(c *check.C) {
 		if err == nil {
 			break
 		}
```

Comment:
so what's the point for sleep in loop to wait?   cause the retry already means fail when call c.Error()